documentation/functionalDoc/functionalDoc.tex

```diff
@@ -232,10 +232,13 @@ \subsection{Static Var Compensators}
 \section{Solver}
 
 The solver used is the Simplified Solver (SolverSIM) from \Dynawo.
-The Maximum time step value is 10s.
+The Maximum time step value is by default 10s. This value can be modified in the configuration file (see section \ref{Dynaflow_Launcher_Configuration_Configuration_File}).
```

Comment:
Maybe we should specify that the value provided should be above 1 (hMin)

Reply:
done, thx